src/test/java/core/basesyntax/imp/FruitServiceImpTest.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.imp;
+
+import core.basesyntax.model.FruitRecordDto;
+import core.basesyntax.service.CustomFileReader;
+import core.basesyntax.service.FruitOperation;
+import core.basesyntax.service.FruitParse;
+import core.basesyntax.service.FruitService;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FruitServiceImpTest {
+    @Test
+    public void makeReportRight_ok() {
```

Comment:
you are doing too much here, this method should only test `fruitService.makeReport`

Reply:
and this?